doc/api/esm.md

```diff
@@ -337,36 +337,42 @@ changes:
       - v14.18.0
     pr-url: https://github.com/nodejs/node/pull/38587
     description: Add support for WHATWG `URL` object to `parentURL` parameter.
+  - version:
+      - REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/49028
+    description: Unflag import.meta.resolve, with `parentURL` parameter still
+                 flagged.
 -->
 
-> Stability: 1 - Experimental
```

Comment:
@ljharb I did remove this, but we can retain it. I'm ok either way.

Reply:
How about you change it to

```
> Stability: 1.2 - Release candidate
```

Per https://nodejs.org/api/documentation.html#stability-index. Then once some time has gone by and we don’t get bug reports, we mark it as fully stable. Maybe with Node 21 in October.

Reply:
I added it.